docs/api-docs/storefront/graphql/graphql-api-orders.mdx

```diff
@@ -0,0 +1,398 @@
+# GraphQL Storefront API: Orders (beta)
+
+<Callout type="info">
+    This feature is currently in beta and is only available to participants. 
+</Callout>
+
+## Overview
+
+BigCommerce's GraphQL Storefront API Orders provides the same access to order objects as the REST Storefront API does. Developers can use frontend GraphQL to create an end-to-end shopper experience. 
+
+This GraphQL API endpoint powers customer solutions as shoppers evolve from guests, to registered customers. Orders related mutations and queries can do the following:
+
+### Get order and details
+
+Get all order details for a specific customer using the `GetOrderDetails` query.
+
+Use `GetOrderDetails` to:
+  - Query for a specific order 
```

Comment:
These are mixed up.  

We need to switch these since "orders" allows you to filter by status, date, and offers pagination along with specifics you might want such as 

"order" allows you to query the specific details.

Reply:
Switching it up with better headers.